trunk/SUAVE/Components/Energy/Networks/Lift_Cruise.py

```diff
@@ -682,16 +933,33 @@ def add_transition_unknowns_and_residuals_to_segment(self, segment, initial_volt
             self.number_of_lift_rotor_engines = int(self.number_of_lift_rotor_engines)
         
         # Setup the residuals
-        segment.state.residuals.network             = Data()        
-        segment.state.residuals.network.voltage     = 0. * ones_row(1)
-        segment.state.residuals.network.propellers  = 0. * ones_row(n_props)
-        segment.state.residuals.network.lift_rotors = 0. * ones_row(n_lift_rotors)
+        segment.state.residuals.network = Data()        
         
-        # Setup the unknowns
-        segment.state.unknowns.throttle_lift                = initial_throttle_lift           * ones_row(1)   
-        segment.state.unknowns.battery_voltage_under_load   = initial_voltage                 * ones_row(1)
-        segment.state.unknowns.propeller_power_coefficient  = initial_prop_power_coefficient  * ones_row(n_props)
-        segment.state.unknowns.lift_rotor_power_coefficient = initial_lift_rotor_power_coefficient * ones_row(n_lift_rotors)
+        if type(self.battery) == SUAVE.Components.Energy.Storages.Batteries.Constant_Mass.Lithium_Ion_LiFePO4_18650: 
```

Comment:
all of my prior comments for battery propeller still hold here

Reply:
addressed